src/main/java/ua/lviv/iot/algo/part1/lab1/StoneWriter.java

```diff
@@ -0,0 +1,54 @@
+package ua.lviv.iot.algo.part1.lab1;
+
+import com.opencsv.CSVWriter;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
+
+public class StoneWriter {
+    public static final File RESULTS = new File("C:/Users/"
+            + "MIHAILO/IdeaProjects/"
+            + "Labs/src/main/resources/results.scv");
```

Comment:
Instead of this path, pls, use this `String path = String.format("%s%s%s%s%s", System.getProperty("user.dir"), File.separator, "src\\main\\resources", File.separator, "result.csv"); `